Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Taxonomy of Failsafe Levels
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+
+## Abstract
+
+When talking about redundancy and backups in the context of cloud infrastructures, the scope under which circumstances these concepts apply to various ressources is neither homogenous nor intuitive.
+This decision record aims to define different levels of failure-safety.
+These levels can then be used in standards to clearly set the scope that certain procedures in e.g. OpenStack offer.
+
+## Terminology
+
+Image
+  OpenStack resource, virtual machine images usually residing in a network storage backend.
+Volume
+  OpenStack resource, virtual drive which usually resides in a network storage backend.
+Virtual Machine (abbr. VM)
+  IaaS resource, also called server, executes workloads from users.
+Secret
+  OpenStack ressource, cryptographic asset stored in the Key Manager (e.g. Barbican).
+Key Encryption Key (abbr. KEK)
+  OpenStack resource, used to encrypt other keys to be able to store them encrypted in a database.
+Floating IP (abbr. FIP)
+  OpenStack resource, an IP that is usually routed and accessible from external networks.
+Disk
+  A physical disk drive (e.g. HDD, SSD) in the infrastructure.
+Node
+  A physical machine in the infrastructure.
+Cyber threat
+  Attacks on the infrastructure through the means of electronic access.
+
```

Comment:
I know, it's bit picky and I like to have the same look and feel in our standards and decision records. I would display terminology as table and call it glossary.

```suggestion
## Glossary
| Term              | Explanation                                                                                                                                                |
| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
| Image           | OpenStack resource, virtual machine images usually residing in a network storage backend.                  |  
Volume           | OpenStack resource, virtual drive which usually resides in a network storage backend.                           | 
| Virtual Machine (abbr. VM) |   IaaS resource, also called server, executes workloads from users.                                   | 
| Secret |   OpenStack ressource, cryptographic asset stored in the Key Manager (e.g. Barbican).                                    | 
| Key Encryption Key (abbr. KEK) |  OpenStack resource, used to encrypt other keys to be able to store them encrypted in a database.                                                                              | 
| Floating IP (abbr. FIP) |   OpenStack resource, an IP that is usually routed and accessible from external networks.       | 
| Disk | A physical disk drive (e.g. HDD, SSD) in the infrastructure.                                                                                    | 
| Node | A physical machine in the infrastructure.                                                                                                               | 
| Cyber threat | Attacks on the infrastructure through the means of electronic access.                                                       | 

```

Reply:
I took Markus table from the PR you mentioned above and adjusted it to comply.

For the title: I took the example DR from the standards repository. Maybe we should change the title from Terminology to Glossary there too.